config.yaml

```diff
@@ -4,11 +4,11 @@
 # user: username to authenticate as
 # password: password for user
 mysql:
-  server: '${MYSQL_HOST}'
-  port: ${MYSQL_PORT}
-  database: '${MYSQL_DATABASE}'
-  user: '${MYSQL_USER}'
-  password: '${MYSQL_PASSWORD}'
+  server: 'mysql'
+  port: 3306
+  database: 'cytube3'
+  user: 'cytube3'
+  password: 'super_secure_password'
   pool-size: 10
```

Comment:
Since you're mounting in the config anyways, would it make sense to ask people to get the config template from upstream so that you don't have to update this one whenever the main repo changes it?  There is also a separate `conf` folder with auxiliary configuration.